src/main/java/winstone/Launcher.java

```diff
@@ -158,7 +158,7 @@ public Launcher(Map args) throws IOException {
 
             int qtpMaxThread = Option.QTP_MAXTHREADS.get(args);
             QueuedThreadPool queuedThreadPool = qtpMaxThread>0?new QueuedThreadPool(qtpMaxThread):new QueuedThreadPool();
-            queuedThreadPool.setName("Jetty Thread Pool");
+            queuedThreadPool.setName("Jetty (winstone)");
```

Comment:
using "Jetty (winstone)"  to match the format in jenkinsci/jenkins-test-harness#197